### PR TITLE
[Alpha 5] Add elmah connector to alpha5 branch

### PR DESCRIPTION
**Pull request by sachinira**
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: openapi/elmah/client.bal [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+        UptimeCheck[] response = check self.clientEp-> get(path, targetType = UptimeCheckArr);
+        return response;
+    }
+}
+
+# Generate query path with query parameter.
+#
+# + queryParam - Query parameter map
+# + return - Returns generated Path or error at failure of client initialization
+isolated function  getPathForQueryParam(map<anydata>   queryParam)  returns  string|error {
```
Comment (abeykoon) [inline openapi/elmah/client.bal:302]: @aneeshafedo why there is an unwanted space between `map<anydata>` and `queryParam`?

Comment (aneeshafedo) [reply openapi/elmah/client.bal:302]: This seems like a bug in the code generation.

Comment (aneeshafedo) [reply openapi/elmah/client.bal:302]: Added an issue https://github.com/ballerina-platform/ballerina-openapi/issues/453

**Comment by abeykoon** [comment]
Merging as this is only a formatting issue. Code will get fixed for beta 2 branch.

**Pull request merged**